src/d3a/models/strategy/pv.py

```diff
@@ -32,25 +31,21 @@
 
 
 class PVStrategy(BaseStrategy):
-    available_triggers = [
-        Trigger('risk', {'new_risk': int},
-                help="Change the risk parameter. Valid values are between 1 and 100.")
-    ]
 
     parameters = ('panel_count', 'initial_selling_rate', 'final_selling_rate',
                   'fit_to_limit', 'update_interval', 'energy_rate_decrease_per_update',
                   'max_panel_power_W')
 
-    def __init__(self, panel_count: int=1,
+    def __init__(self, panel_count: int = 1,
                  initial_selling_rate:
-                 float=ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE,
+                 float = ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE,
                  final_selling_rate:
-                 float=ConstSettings.PVSettings.FINAL_SELLING_RATE,
-                 fit_to_limit: bool=True,
+                 float = ConstSettings.PVSettings.FINAL_SELLING_RATE,
+                 fit_to_limit: bool = True,
                  update_interval=duration(minutes=ConstSettings.GeneralSettings.UPDATE_RATE),
                  energy_rate_decrease_per_update:
-                 float=ConstSettings.GeneralSettings.ENERGY_RATE_DECREASE_PER_UPDATE,
-                 max_panel_power_W: float=None):
+                 float = ConstSettings.GeneralSettings.ENERGY_RATE_DECREASE_PER_UPDATE,
+                 max_panel_power_W: float = None):
```

Comment:
Sorry, I didn't get it. Are there any specific rules to add space in between `=` while defining default values?

Reply:
Yes. Pycharm also plains about that. It is not set as crucial in flake8. It is minor indeed.

Reply:
Now, that I noticed, Pycharm is also suggesting to have classes arguments to be lower case. For Eg: [https://github.com/gridsingularity/d3a/blob/master/src/d3a/models/state.py#L69](url)